runtime-modules/forum/src/lib.rs

```diff
@@ -461,18 +447,15 @@ pub struct Poll<Timestamp, Hash> {
 /// Represents a thread post
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
 #[derive(Encode, Decode, Default, Clone, PartialEq, Eq)]
-pub struct Post<ForumUserId, ModeratorId, ThreadId, BlockNumber, Moment, Hash> {
+pub struct Post<ForumUserId, ModeratorId, ThreadId, Hash> {
     /// Id of thread to which this post corresponds.
     pub thread_id: ThreadId,
 
     /// Unique identifier
     pub hash: Hash,
 
     /// Possible moderation of this post
-    pub moderation: Option<ModerationAction<ModeratorId, BlockNumber, Moment>>,
-
-    /// When post was submitted.
-    pub created_at: BlockchainTimestamp<BlockNumber, Moment>,
+    pub moderation: Option<ModerationAction<ModeratorId>>,
```

Comment:
Why is this still here?

Reply:
We discussed complete removal of moderation after I created PRs *forum archival state* 3 and 4. To prevent merge conflicts in 4, `moderationAction` will be removed in additional PR with a couple of other changes.